web/packages/teleterm/notarize.js

```diff
@@ -1,4 +1,4 @@
-const { notarize } = require('electron-notarize');
+const { notarize } = require('@electron/notarize');
```

Comment:
electron-notarize is deprecated in favor of @electron/notarize. Fortunately, it's just a package name change. dependabot will make sure that it stays up to date going forward.